### PR TITLE
fix pybind11 high-level binds

### PR DESCRIPTION
**Pull request by andreaostuni**
This PR solves the problem mentioned in #1127

### CODE REVIEW

File: src/high-level/CMakeLists.txt [hunk 1]
```diff
@@ -28,8 +28,8 @@ install(TARGETS ${libraryname}
         RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}" COMPONENT bin
         LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT shlib
         ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}" COMPONENT lib
-        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/iDynTree/
-        PRIVATE_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/iDynTree/)
+        PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/iDynTree
+        PRIVATE_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/iDynTree/impl)
```
Comment (traversaro) [inline src/high-level/CMakeLists.txt:32]: Just a curiosity, did you modified this for consistency with the other components?

**Comment by traversaro** [comment]
Thanks @andreaostuni ! Just a curiosity, why are you using the pybind11-bindings instead of the swig-based ones? Do you need a release "quick" with this changes?

**Comment by traversaro** [comment]
fyi @andreaostuni if you have any opinion.

**Comment by francesco-romano** [comment]
> fyi @francesco-romano if you have any opinion.

Opinion on... what?

**Comment by traversaro** [comment]
> > fyi @francesco-romano if you have any opinion.
> 
> Opinion on... what?

Sorry, on the PR!

**Review by traversaro — APPROVED** [review]

**Comment by andreaostuni** [comment]
> Thanks @andreaostuni ! Just a curiosity, why are you using the pybind11-bindings instead of the swig-based ones? Do you need a release "quick" with this changes?

I'm actually new in the world of bindings and I don't know the difference really well and since I had already had pybind I tried with it!

**Comment by traversaro** [comment]
I see, thanks! Just a word of warning: most of Python code for iDynTree you may find around (or that LLMs models generate) is probably oriented toward the swig bindings of iDynTree.

**Pull request merged**